test/unit/sources/alternates/deviantart_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'test_helper'
```

Comment:
These test cases would benefit from a shared assert function. I have something in mind like danbooru does: https://github.com/danbooru/danbooru/blob/master/test/unit/sources/deviant_art_test.rb
https://github.com/danbooru/danbooru/blob/9cab67c0ac72a8c52289302c519715ceec2372d9/test/test_helpers/source_test_helper.rb#L8

This obviously does more than what is needed here, but should get my point across.

Reply:
You could pass the description to the method, like 'alternate_should("remove tracking data", ...)`.

The more correct place for this would be in the helper directory, and then include the module into the correct test type in the global `test_helper.rb`. I'm not that big on ruby/rails tests myself, so I can mostly just parrot what I saw in this repo.